R/uvhydrograph-render.R

```diff
@@ -344,10 +344,18 @@ createSecondaryPlot <- function(uvInfo, secondarySeriesList,
   
   if(!isEmptyOrBlank(secondarySeriesList[['corrected']][['points']][['value']])){
     lims <- calculateLims(secondarySeriesList[['corrected']][['points']])
-    corrLimPoints <- secondarySeriesList[['corrected']][['points']][['value']]
```

Comment:
These weren't used, maybe that's part of the problem...  ? I removed them but perhaps adding them into logic below is the correct solution rather than what I've worked up here.